crates/nq-packetloss/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "nq-packetloss"
+version = { workspace = true }
+edition = { workspace = true }
+repository = { workspace = true }
+authors = { workspace = true }
+license = { workspace = true }
+readme = "../README.md"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+nq-core = { workspace = true }
+nq-load-generator = { workspace = true }
+nq-tokio-network = { workspace = true }
+
+anyhow = { workspace = true, features = ["backtrace"] }
+http = { workspace = true }
+http-body-util = { workspace = true }
+hyper = { workspace = true }
+rustls = { version = "0.23.10", default-features = false, features = ["std", "ring"] }
+serde = { workspace = true, features = ["derive"] }
+serde_json = { workspace = true }
+tokio = { workspace = true }
+tokio-util = { workspace = true }
+tracing = { workspace = true }
+url = { workspace = true, features = ["serde"] }
+webrtc = "0.12.0"
```

Comment:
nit: can we add this to the root `Cargo.toml` and pull it in as a workspace dep?